static/app/components/events/autofix/autofixChanges.analytics.spec.tsx

```diff
@@ -35,6 +42,24 @@ describe('AutofixChanges', () => {
   beforeEach(() => {
     MockApiClient.clearMockResponses();
     mockButton.mockClear();
+    jest.mocked(useAutofixRepos).mockReset();
+    jest.mocked(useAutofixData).mockReset();
+    jest.mocked(useAutofixRepos).mockReturnValue({
+      repos: [],
+      codebases: {},
+    });
+    jest.mocked(useAutofixData).mockReturnValue({
+      data: {
+        request: {
+          repos: [],
+        },
+        codebases: {},
+        created_at: '2024-01-01T00:00:00Z',
+        run_id: '456',
+        status: AutofixStatus.COMPLETED,
+      },
+      isPending: false,
+    });
```

Comment:
isn't it better to mock API responses rather than hooks?

Reply:
nvm i remember why I did it this way, the mock api response doesn't work because `useAutofixData` is `enabled:false` and only fetches from cache, mocking that would be difficult